src/random_test.cpp

```diff
@@ -2557,13 +2557,14 @@ void alter_tablespace_rename(Thd1 *thd) {
 static std::vector<std::string> load_special_sql_from() {
   std::vector<std::string> array;
   auto file = opt_string(SQL_FILE);
-  std::string line;
+  std::string sql;
 
   std::ifstream myfile(file);
   if (myfile.is_open()) {
     while (!myfile.eof()) {
-      getline(myfile, line);
-      array.push_back(line);
+      getline(myfile, sql);
+      if (sql.size() > 0)
```

Comment:
If the string sql just has a blank space, it will still be treated as a SQL statement. Maybe add a check to ignore lines with only blank spaces

Reply:
fixed.